druid/src/bloom.rs

```diff
@@ -0,0 +1,158 @@
+// Copyright 2020 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! A simple bloom filter, used to track child widgets.
+
+use std::hash::{Hash, Hasher};
+use std::marker::PhantomData;
+
+use fnv::FnvHasher;
```

Comment:
This is *so* something that can be tuned later, but I was thinking of SplitMix64.